packages/core-transactions/src/handlers/delegate-registration.ts

```diff
@@ -110,7 +110,7 @@ export class DelegateRegistrationTransactionHandler extends TransactionHandler {
         return true;
     }
 
-    protected applyToSender(transaction: Interfaces.ITransaction, walletManager: State.IWalletManager): void {
+    public applyToSender(transaction: Interfaces.ITransaction, walletManager: State.IWalletManager): void {
```

Comment:
Don't make those public. Just keep the ones you removed.

Reply:
It makes no sense to have:
```
class A {
  public method1(x, y) {
    this.method2(x, y);
  }
  private method2(x, y) {
    ...
  }
}
```

Reply:
It makes sense when taking into account that it is not supposed to be called from the outside except by the pool. Now it gives the impression it's okay to call it from anywhere. Whatever, I don't insist on it.